crates/module-system/sov-modules-api/src/transaction/mod.rs

```diff
@@ -379,6 +380,11 @@ impl<R: TransactionCallable, S: Spec> UnsignedTransaction<R, S> {
             self.details,
         )
     }
+
+    /// Returns a copy of the RuntimeCall from this unsigned transaction.
+    pub fn call(&self) -> R::Call {
```

Comment:
Looks like thi method is unused

Reply:
I do use it, in `decode_solana_json_tx()`.